site/examples/check-lists.js

```diff
@@ -32,7 +32,45 @@ const CheckListsExample = () => {
 }
 
 const withChecklists = editor => {
-  const { deleteBackward } = editor
+  const { deleteBackward, insertBreak } = editor
+  editor.insertBreak = () => {
+    const match = Editor.above(editor, {
+      match: n => {
+        return Editor.isBlock(editor, n)
+      },
+    })
+    if (match) {
+      const [matchingNode] = match
+      if (matchingNode.type !== 'paragraph') {
+        if (
+          matchingNode.children[matchingNode.children.length - 1].text
+            .length === 0
+        ) {
+          Transforms.delete(editor)
+          const paragraph = {
+            type: 'paragraph',
+            children: [{ text: '' }],
+          }
+          Transforms.insertNodes(editor, paragraph)
+          Transforms.unwrapNodes(editor, {
+            match: n => n.type === 'check-list-item',
+            split: true,
+          })
```

Comment:
This doesn't do anything as you just removed the check-list-item above, there is no "parent" check-list-item to unwrap?

Reply:
True. Let me update and test it once and I'll update the pull request

Reply:
@zbeyens I tested the code with the suggested change and there were no issues with removing it. I have updated my pull request to reflect the same.